README.md

```diff
@@ -985,6 +1036,29 @@ Which, when called, returns this string:
 
 <br />
 
+## Serialization table
+
+Here is the table that explains how this library handles different types of children,
+describing the inputs and outputs.
+
+> [!IMPORTANT]  
+> Arrays are simply concatenated with no separator.
+
+| Input Type                 | Output Type           |
+| -------------------------- | --------------------- |
+| `<div>{"abc"}</div>`       | `<div>abc</div>`      |
+| `<div>{10}</div>`          | `<div>10</div>`       |
+| `<div>{NaN}</div>`         | `<div>NaN</div>`      |
+| `<div>{Infinity}</div>`    | `<div>Infinity</div>` |
+| `<div>{true}</div>`        | `<div>true</div>`     |
+| `<div>{false}</div>`       | `<div>false</div>`    |
+| `<div>{null}</div>`        | `<div></div>`         |
+| `<div>{undefined}</div>`   | `<div></div>`         |
+| `<div>{[1, 2, 3]}</div>`   | `<div>12</div>`       |
```

Comment:
the output here seems off, or at least odd?

Reply:
thanks!